components/usage/pkg/db/workspace_test.go

```diff
@@ -90,8 +90,8 @@ func TestListWorkspacesByID(t *testing.T) {
 	conn := db.ConnectForTests(t)
 
 	workspaces := []db.Workspace{
-		dbtest.NewWorkspace(t, db.Workspace{ID: "gitpodio-gitpod-aaaaaaaaaaa"}),
-		dbtest.NewWorkspace(t, db.Workspace{ID: "gitpodio-gitpod-bbbbbbbbbbb"}),
+		dbtest.NewWorkspace(t, db.Workspace{}),
```

Comment:
The change here is that it generates random IDs, rather than keeping them fixed. This doesn't matter as we reference the actually created IDs rather than the fixed values.